VTTiny.Plugin.Discord/Services/Token.cs

```diff
@@ -0,0 +1,150 @@
+﻿using System.Runtime.InteropServices;
+using System.Runtime.Serialization;
+using System.Text.Json;
+
+namespace VTTiny.Plugin.Discord.Services;
+
+public class Token
+{
+    public Token()
+    {
+        
+    }
+    public string? token
+    {
+        get; set;
+    }
+
+    public string getToken()
+    {
+        BotPaths.CreateFoldersAndFiles();
+        Console.WriteLine("Getting Token");
+        try
+        {
+            Console.WriteLine("Deserializing Token");
+            //Read the token from the json file
+            
+            var tk = JsonSerializer.Deserialize<Token>(File.ReadAllText(BotPaths.token));
+            Console.WriteLine("Deserialized Token");
+            token = tk.token;
+            Console.WriteLine("Got Token from Json");
+            //if this is windows
+            if (string.IsNullOrEmpty(token))
+            {
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                {
+                    //Makes a popup window that says "You are an Idiot, you forgot the Token! {BotPaths.token}"
+                    string msg =
+                        """powershell -WindowStyle hidden -Command "& {[System.Reflection.Assembly]::LoadWithPartialName('System.Windows.Forms'); [System.Windows.Forms.MessageBox]::Show('You forgot to put a token in, The folder is in %appdata%/VTtinyDiscordBot/Token.json','WARNING')} """;
+                    var process = System.Diagnostics.Process.Start("cmd.exe", msg);
+                    process.WaitForExit();
+                    Environment.Exit(0);
+                }
+            }
+            if (string.IsNullOrEmpty(token))
+                throw new IdiotException($"You are an Idiot, you forgot the Token! {BotPaths.token}");
```

Comment:
I know that this is a WIP, but please don't be rude to the user :). A more suitable name would be `TokenNotFoundException`.

Reply:
oh i forgot, i copy pasted most code from a previous bot i've made

Reply:
10/10 reusing an originally only for internal usage error